applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -197,34 +174,6 @@ static struct bt_bap_stream_ops stream_ops = {
 #endif /* (CONFIG_BT_AUDIO_RX) */
 };
 
-static bool scan_recv_cb(const struct bt_le_scan_recv_info *info, struct net_buf_simple *ad,
-			 uint32_t broadcast_id)
-{
-	char name[MAX(sizeof(CONFIG_BT_AUDIO_BROADCAST_NAME),
-		      sizeof(CONFIG_BT_AUDIO_BROADCAST_NAME_ALT))] = {'\0'};
-	struct bt_name bis_name = {&name[0], ARRAY_SIZE(name)};
-
-	bt_data_parse(ad, adv_data_parse, (void *)&bis_name);
-
-	if (strlen(bis_name.name) ==
-	    strlen(brdcast_src_names[active_stream.brdcast_src_name_idx])) {
-		if (strncmp(bis_name.name, brdcast_src_names[active_stream.brdcast_src_name_idx],
-			    strlen(brdcast_src_names[active_stream.brdcast_src_name_idx])) == 0) {
-			LOG_INF("Broadcast source %s found", bis_name.name);
-			return true;
-		}
-	}
-
-	return false;
-}
-
-static void scan_term_cb(int err)
```

Comment:
We didn't use it for anything, other than print. I can add it back

Reply:
After looking at it a bit moreI found this in the documentation: 
`Scan` terminated callback is called whenever a scan started by bt_bap_broadcast_sink_scan_start() is terminated before bt_bap_broadcast_sink_scan_stop()`
So since we don't ut sink_scan_start, we shouldn't use scan_termintated either

Reply:
Hmm the bt_mgmt files do not know if a PA sync has been lost now then, if I am not missing something?

Reply:
Correct, that is only known by the le_audio files. I couldn't find a callback to connect the scanning and syncing to a PA with the termination of it. Maybe @Thalley can shed some light on that

Reply:
There's the `bt_le_per_adv_sync_cb.term` callback if that is what you are thinking about? Not sure I completely follow this comment

Reply:
Thank you, then I could use bt_le_per_adv_sync_cb_register in the scan_for_broadcast. Might need a word with you about this to make sure this will work as intended. I'm especially wondering about the flow in the broadcast_sink if the callbacks for pa_sync is somewhere else. You talked about the PA_sync and the BIS uncoupled, but is that possible at the moment?

Reply:
Keep as is, after discussing with Emil